### PR TITLE
FE card-6 (delete appointment)

### PR DESCRIPTION
**Pull request by KuzminAanton**

### CODE REVIEW

File: src/components/Main.jsx [hunk 1]
```diff
@@ -1,16 +1,301 @@
...
+            <div className="main-content-sort">
+              <div className="main-content-sort__wrapper">
+                <div className="main-content-sort-input">
+                  <span>Сортировать по:</span>
+                  <Select
+                    className="main-content-sort-input-fields"
+                    value={checkMainFilter === 'empty' ? '' : checkMainFilter}
+                    onChange={(e) => setCheckMainFilter(e.target.value)}
+                  >
+                    <MenuItem value="empty">Не выбрано</MenuItem>
```
Comment (usatovaDasha) [inline src/components/Main.jsx:176]: map

File: src/elements/ModalWindow.jsx [hunk 2]
```diff
@@ -0,0 +1,224 @@
+import React, { useContext, useEffect, useState } from 'react';
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx:1]: imports style

File: src/elements/ModalWindow.jsx [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+              </div>
+            </Box>
+          </Fade>
+        </Modal>
+      </div>
+    );
+  }
+
+  return (
+    <div>
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx:184]: in component

File: src/routes/Auth.jsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
+import React, { useContext, useState } from 'react';
```
Comment (usatovaDasha) [inline src/routes/Auth.jsx:1]: import style

File: src/elements/Header.jsx [hunk 5]
```diff
@@ -1,23 +1,31 @@
-import React from 'react';
-
+import React, { useContext } from 'react';
```
Comment (usatovaDasha) [inline src/elements/Header.jsx]: imports style

File: src/elements/Header.jsx [hunk 6]
```diff
@@ -1,23 +1,31 @@
...
+      <div className="header">
+        <div className="header-title">
+          <div className="header-title__logo">
+            <img src="../images/icon-logo.svg" alt="logo" />
+          </div>
+          <div className="header-title__text">
+            <p>{headerParam.text}</p>
+          </div>
         </div>
+        {headerParam.checkBtn
```
Comment (usatovaDasha) [inline src/elements/Header.jsx]: use &&

File: src/elements/Header.scss [hunk 7]
```diff
@@ -1,3 +1,12 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  max-width: 989px;
```
Comment (usatovaDasha) [inline src/elements/Header.scss]: on full screen

File: src/elements/ModalWindow.jsx (hunk 2 above)
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx]: move to single lines

File: src/elements/ModalWindow.jsx [hunk 8]
```diff
@@ -0,0 +1,224 @@
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Backdrop,
+  Box,
+  Button,
+  Fade, FormControl, MenuItem,
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx]: higher

File: src/elements/ModalWindow.jsx [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+  Modal,
+  Select,
+  TextField,
+} from '@mui/material';
+import axios from 'axios';
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx]: imports style

File: src/elements/ModalWindow.jsx [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+      date,
+      complaints,
+    },
+    {
+      headers: {
+        'Content-Type': 'application/json',
+        authorization: token,
+      },
+    }).then((res) => {
+      if (!res.data.error) {
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx]: desctructurization

File: src/elements/ModalWindow.jsx [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+
+  const deleteAppointment = async () => {
+    await axios.delete(`http://localhost:5000/deleteAppointments?_id=${_idList}`,
+      {
+        headers: {
+          'Content-Type': 'application/json',
+          authorization: token,
+        },
+      }).then((res) => {
+      if (!res.data.error) {
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx:89]: destructurization

File: src/elements/ModalWindow.jsx [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+        setAppointmentList([...res.data.data]);
+      }
+    });
+    handleCloseModal();
+  };
+
+  if (typeModal === 'edit') {
+    return (
+      <div>
+        <Button onClick={handleOpenModal}>
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx:99]: correct call function

File: src/elements/ModalWindow.jsx [hunk 13]
```diff
@@ -0,0 +1,224 @@
...
+              <div className="modal-edit">
+                <div className="modal-edit-header">
+                  <h3>Изменить прием</h3>
+                </div>
+                <FormControl>
+                  <div className="modal-edit-form">
+                    <div className="modal-edit-form__input-elem">
+                      <p>Имя:</p>
+                      <TextField
+                        value={inputValueEdit.name}
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx:124]: desctructurization

File: src/elements/ModalWindow.jsx [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+      if (!res.data.error) {
+        setAppointmentList([...res.data.data]);
+      }
+    });
+    handleCloseModal();
+  };
+
+  if (typeModal === 'edit') {
+    return (
+      <div>
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx:98]: in component

File: src/elements/ModalWindow.jsx (hunk 3 above)
Comment (usatovaDasha) [reply src/elements/ModalWindow.jsx:186]: !!!!!

File: src/elements/ModalWindow.jsx [hunk 15]
```diff
@@ -0,0 +1,224 @@
...
+            </Box>
+          </Fade>
+        </Modal>
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      <Button onClick={handleOpenModal}>
```
Comment (usatovaDasha) [inline src/elements/ModalWindow.jsx:187]: correct call function

File: src/elements/ModalWindows.scss [hunk 16]
```diff
@@ -0,0 +1,9 @@
+.modal-ed {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 400px;
```
Comment (usatovaDasha) [inline src/elements/ModalWindows.scss]: almost full screen

File: src/routes/Auth.jsx [hunk 17]
```diff
@@ -0,0 +1,122 @@
...
+  const { loginValue, passwordValue, retryPasswordValue } = inputState;
+  const [errorAlertLog, setErrorAlertLog] = useState(false);
+  const [errorAlertPass, setErrorAlertPass] = useState(false);
+  const [errorAlertPassReplay, setErrorAlertPassReplay] = useState(false);
+  const [errorText, setErrorText] = useState('');
+  const { login } = useContext(AuthContext);
+
+  const formSubmit = (e, flag) => {
+    if (!(loginValue || passwordValue || retryPasswordValue)) {
+      setErrorText('Введите данные');
```
Comment (usatovaDasha) [inline src/routes/Auth.jsx]: use spread

File: src/routes/Auth.jsx [hunk 18]
```diff
@@ -0,0 +1,122 @@
...
+                });
+              }
+              break;
+            case 'log':
+              axios.post('http://localhost:5000/auth/login', {
+                login: loginValue,
+                password: passwordValue,
+              }).then((res) => {
+                if (res.data) {
+                  login(res.data.token, res.data.userId);
```
Comment (usatovaDasha) [inline src/routes/Auth.jsx]: the user ID should not come from the server at all

File: src/components/Main.jsx [hunk 19]
```diff
@@ -1,16 +1,301 @@
...
+                            <img src="../images/icon-delete-big.svg" alt="" />
+                          </div>
+                        </Button>
+                      </div>
+                    </div>
+                  )
+                  : <></>
+              }
+            </div>
+            <div className="main-content-table">
```
[the remark this answers is not in the document]
Comment (KuzminAanton) [reply src/components/Main.jsx]: this is name of class. `table` - structure, looks like this. this is rendered with `map` function

File: src/elements/Header.scss (hunk 7 above)
Comment (KuzminAanton) [reply src/elements/Header.scss]: on full screen - i have this changes in later pull requests

**Pull request merged**